hail/python/hail/backend/service_backend.py

```diff
@@ -334,7 +344,11 @@ async def _rpc(self,
                         iodir + '/out'
                     ],
                     mount_tokens=True,
-                    resources={'preemptible': False, 'memory': 'standard'}
+                    resources={
+                        'preemptible': False,
+                        'cpu': self.driver_cores or '1',
```

Comment:
Can you double check what the type of 'cpu' and memory needs to be? I think we need to cast them as `str`.